script.js

```diff
@@ -0,0 +1,487 @@
+let timer;
+
+function getLangFromStorage() {
+  try {
+    storageValue = localStorage.getItem('lang');
+  } catch (err) {
+    isStorageSupport = false;
+  }
+}
+
+getLangFromStorage();
+
+let lang = storageValue || 'eng';
+const ROWS_SIZE = [14, 29, 42, 55, 64];
+const MULTI_CONTENT_KEYS = 12;
+const LANG_CHANGING_KEYS = [['Alt', 'Shift'], ['AltGraph', 'Shift']];
+const replacedTextKeys = [['Delete', 'Del'], ['CapsLock', 'CapsLk'], ['Control', 'Ctrl'], ['Meta', 'Win'], ['ArrowUp', 'Up'], ['Backspace', 'Back'], ['AltGraph', 'Alt'], ['ArrowDown', 'Down'], ['ArrowLeft', 'Left'], ['ArrowRight', 'Right']];
+const doubledKeysList = ['Alt', 'AltGraph', 'Control', 'Shift'];
+
+const lettersList = {
+  rusKeys: [['ё', 'Ё'], ['1', '!'], ['2', '"'], ['3', '№'], ['4', '; '], ['5', ' % '], ['6', ':'], ['7', '?'], ['8', '*'], ['9', '('], ['0', ')'], ['-', '_'], ['=', '+'], 'Backspace', 'Tab', 'й', 'ц', 'у', 'к', 'е', 'н', 'г', 'ш', 'щ', 'з', 'х', 'ъ', '\\', 'Delete', 'CapsLock', 'ф', 'ы', 'в', 'а', 'п', 'р', 'о', 'л', 'д', 'ж', 'э', 'Enter', 'Shift', 'я', 'ч', 'с', 'м', 'и', 'т', 'ь', 'б', 'ю', '.', 'ArrowUp', 'Shift', 'Control', 'Meta', 'Alt', ' ', 'AltGraph', 'ArrowLeft', 'ArrowDown', 'ArrowRight', 'Control'],
+  engKeys: [['`', '~'], ['1', '!'], ['2', '@'], ['3', '#'], ['4', '$'], ['5', '%'], ['6', '^'], ['7', '&'], ['8', '*'], ['9', '('], ['0', ')'], ['-', '_'], ['=', '+'], 'Backspace', 'Tab', 'q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p', '[', ']', '\\', 'Delete', 'CapsLock', 'a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l', '; ', "'", 'Enter', 'Shift', 'z', 'x', 'c', 'v', 'b', 'n', 'm', ',', '.', '/', 'ArrowUp', 'Shift', 'Control', 'Meta', 'Alt', ' ', 'Alt', 'ArrowLeft', 'ArrowDown', 'ArrowRight', 'Control'],
+};
+
+let templateElem = '';
+let textInput;
+let isCapsOn = false;
+let keyElementsList = [];
+let capslockKey;
+let pressedKeysList = [];
+let keyboardElem;
+
+function getReplacingText(letter) {
+  const replacedKeysArr = replacedTextKeys.flat();
+  const positionInArray = replacedKeysArr.findIndex((replacedKeyItem, index) => {
+    if (index % 2 !== 0) {
+      return false;
+    }
+    if (replacedKeyItem === letter) {
+      return true;
+    }
+    return false;
+  });
+  if (positionInArray !== -1) {
+    return replacedKeysArr[positionInArray + 1];
+  }
+  return false;
+}
+
+function getMupkupFromLetter(letter) {
+  if (typeof letter === 'object') {
+    return `<span class="key">${letter[0]}</span>`;
+  }
+  if (letter === ' ') {
+    return `<span class="key extra-wide-key">${letter}</span>`;
+  }
+  if (letter === 'CapsLock') {
+    return `<span class="key caps-key">${letter}</span>`;
+  }
+  if (letter.indexOf('Arrow') !== -1) {
+    return `<span class='key-wrap'>
+            <span class="key hide-text">${letter}</span>
+            <span class="replacing-text arrow-key arrow-${getReplacingText(letter).toLowerCase()}">${getReplacingText(letter)}</span>
+            </span>`;
+  }
+  if (getReplacingText(letter) !== false) {
```

Comment:
I think this condition is overhead. getReplacingText function returns you true of false. You can write just like this. 

if (getReplacingText(letter)) { ... }